charts/payaza/templates/deployment.yaml

```diff
@@ -35,6 +35,22 @@ spec:
           - name: {{ .Values.deployment.name | quote }}
             mountPath: {{ .Values.volume.path }}
           {{- end }}
+          {{- if .Values.probes.enableProbes }}
+          livenessProbe:
```

Comment:
i have made some changes to enable each probe separately so each one can be tested distinctly